ci/kubetest/helpers/clickhouse.py

```diff
@@ -19,7 +19,7 @@ def get_clickhouse_statefulset_spec(kube):
 
 def get_clickhouse_cluster_service_spec(kube):
     start = time.time()
-    while time.time() - start < 120:
+    while time.time() - start < 300:
```

Comment:
```suggestion
    while (time.time() - start) < 300:
```
I'm just neurotic about clarity on order of ops.